try {
    throw 'ERROR!'
} catch(e) {
    puts e
}
